app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
```

Comment:
- [ ] As stated before use `belongs_to :author, class_name: 'User'` instead of `belongs_to :user`